src/main/java/core/basesyntax/service/ReportService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface ReportService {
+    String getReport();
```

Comment:
minor: I would say generateReport will be a more proper name because get... name suggests that it may already be created